lib/kubernetes-deploy/options_helper.rb

```diff
@@ -2,11 +2,22 @@
 
 module KubernetesDeploy
   module OptionsHelper
-    def self.default_and_check_template_dir(template_dir)
+    def self.default_and_check_template_dir(template_dir, use_stdin: false)
+      if use_stdin
+        if $stdin.tty?
+          puts "Nothing to read from $stdin"
+          exit(1)
```

Comment:
I know this pattern exists elsewhere in this file, but conceptually, I find it is better to reserve `exit` calls for the code that is for sure being invoked from the command line, i.e. our `exe` files. Helpers like this should instead raise exceptions.